content/references/protocol-reference/ledger-data/ledger-object-types/bridge.md

```diff
@@ -1,6 +1,6 @@
 html: bridge.html
 parent: ledger-object-types.html
-blurb: A bridge connects and enables value to move efficiently between two blockchains. 
+blurb: A `bridge` object represents a single cross-chain bridge that connects and enables value to move efficiently between two blockchains. 
```

Comment:
Blurbs should not have markdown in them since they are used in some places (<meta> tags for example) that don't have formatting